plugin/trino-memory/src/test/java/io/trino/plugin/memory/TestMemorySmoke.java

```diff
@@ -150,6 +150,21 @@ public void testCustomMetricsScanOnly()
                 .getMetrics();
     }
 
+    @Test(timeOut = 30_000)
+    public void testPhysicalInputPositions()
```

Comment:
Could you add similar test for ORC in Hive connector?

Reply:
that's tricky to test, requires hive3 setup for acid and assertions on operator stats outside of memory connector are quite flaky